### DISCUSSION TITLE
Microsoft.Graph.Core

### DISCUSSION THREAD

**Discussion by lesjtay** [pnp/powershell#495] [Q&A] [answered]
Below is an error I've tried to solve for a couple of hours now.

New-PnPSite: Could not load file or assembly 'Microsoft.Graph.Core, Version=1.22.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'

nightly v.1.4.15

**Comment by veronicageek** [comment, marked as answer]
Hi, I've no issue on my side 😐

![image](https://user-images.githubusercontent.com/25181757/112623684-8f5aee00-8e24-11eb-8ee0-a4ad8831319c.png)

**Reply by lesjtay** [reply]
Just ready to delete my PS install - left VSCode PowerShell Integrated and tried a straight PS from the same installed PS exe and it worked except it asked for a Connect-PnPOnline. Curious because the PowerShell Integrated claimed to be Connect(ed)-PnPOnline already. Then I tried pwsh in a VSCode terminal again it asked to be Connect(ed) and it ran the NewPnPSite without complaint.

What did they do to the Integrated terminal to make it hate me? Does everyone else know to avoid it or is there some trick that I missed?

**Reply by lesjtay** [reply]
Hi Veronique,

You carried me to the answer by demonstration that it should work (very important). So I marked your test as answer.

Thank you,

Les

**Reply by AndersRask** [reply]
It sometimes help to force authentication with the -ForceAuthentication in connection with -Interactive or -UseWebLogin if credentials have changed for example

**Reply by AndersRask** [reply]
You should also check in VSCode what version you are running in the terminal. It could be that it was using PowerShell 5.1 and the module was installed in PowerShell 7.1 (or vice versa) and hence triggered the error as the module path are different
